docs/modules/general/pages/architecture-overview/remoteComponents.adoc

```diff
@@ -11,10 +11,10 @@ The concept of Remote Components extends the concept of Microfrontends by focusi
 == Remote Components value
 The implementation details of Remote Component are located in a specific Microfrontend's source code. The client side of that Remote Component does not need to prepare additional technical code, which means that they are implemented once and maintained in a single place. With Remote Components its easy to share the technical implementation details from one Application to another.
 
-For example, Remote Components might come in handy when your Applications require an ability to change current user's password via a button. In that scenario Microfrontend responsible for user management would expose a Remote Component that implements such functionality and other Applications with that requirement would use it in the desired place to allow users to change their password. Technical details on how the password change is handled would be in a single place and all Applications using the Remote Component will not need to take care of that matter.
+For example, Remote Components might come in handy when your Applications require an ability to change current user's password via a button. In that scenario user management UI would expose a Remote Component that implements such functionality and other Applications with that requirement would use it in the desired place to allow users to change their password. Technical details on how the password change is handled would be in a single place and all Applications using the Remote Component will not need to take care of that matter.
 
 == OneCX Slot
-Every Remote Component requires a host to be displayed within a page. OneCX Slots are the elements fulfilling this role. To use Remote Component is necessary to first implement and expose it via a Microfrontend remote entry file and then create and confiure OneCX Slot in which it should be displayed.
+Every Remote Component requires a host to be displayed within a page. OneCX Slots are the elements fulfilling this role. To use Remote Component it is necessary to first implement and expose it via a Microfrontend remote entry file and then create and confiure OneCX Slot in which it should be displayed.
```

Comment:
To use **a** Remote component ...
... and then create and **configure a** OneCX Slot ...